ixnetwork_open_traffic_generator/vport.py

```diff
@@ -615,8 +615,19 @@ def results(self, request):
             self._column_names = []
         else:
             self._column_names = request.column_names
+        
+        port_filter = {'property': 'name', 'regex': '.*'}
+        port_names = [port.name for port in self._api._config.ports]
+        if request is not None and request.port_names is not None \
```

Comment:
If we want to be consistent with explicit checks, we should do:
```python
if request is not None and request.port_names is not None and len(request.port_names) > 0:
```

Or if we prefer something terse and implicit, we should do:
```python
if request and request.port_names:
```

Reply:
Actually we coded that way though I don't have any preference. 
Note: Zen of Python said "Explicit is better than implicit".  :)

Reply:
add implicit check